envoy/stream_info/stream_info.h

```diff
@@ -565,6 +575,16 @@ class StreamInfo {
    */
   virtual uint64_t bytesReceived() const PURE;
 
+  /**
+   * @return the stream's state.
+   */
+  virtual absl::optional<StreamState> streamState() const PURE;
+
+  /**
+   * @param protocol the stream's state.
+   */
+  virtual void streamState(StreamState stream_state) PURE;
```

Comment:
setStreamState, please.

Reply:
I followed the same pattern as "protocol" method, I thought this was some kind of convention.